src/commands/positive.ts

```diff
@@ -92,6 +101,44 @@ export default class Positive extends ApiKeyCommand {
     }
   };
 
+  setTopSecurity = async (flags: {
+    apiKey: string | undefined;
+  }): Promise<void> => {
+    const securitySchemes = await this.schema.getSecuritySchemes();
+    const topSecurities = await this.schema.getTopSecurities();
+    const securityTypes = {};
+    for (const scheme of securitySchemes) {
+      securityTypes[OASSecurityType.APIKEY] =
+        this.getAPISecurityKey(scheme, flags) ??
+        securityTypes[OASSecurityType.APIKEY];
```

Comment:
I think we lose some clarity behind our intentions using `??` instead of an if statement here. 
what do you think about separating out the security failures part from `getAPISecurityKey` and just doing
```ts
for (const scheme of securitySchemes) {
    if (scheme.securityType === OASSecurityType.APIKEY) {
        securityTypes[OASSecurityType.APIKEY] = schema.name
    }
}
```
We could then do something like
```ts
if (securityTypes[OASSecurityType.APIKEY] && !flags.apiKey) {
    flags.apiKey = this.promptUser('what is your apiKey?', {type: mask}) // this would just be a wrapper method around the cli prompt method
}
```